assembly/common/src/main/resources/bin/sqlshell.sh

```diff
@@ -149,22 +149,19 @@ if [[ "$jversion" < "1.8" ]]; then
    exit 1
 fi
 
-# set splice lib dir
-SPLICE_LIB_DIR="##SPLICELIBDIR##"
```

Comment:
removing this will break the on-premise install for cloudera, hortonworks and mapr

Reply:
there is a part of the assembly pom which replaces this token ##SPLICELIBDIR## with the appropriate directory, such as /opt/cloudera/parcels/SPLICEMACHINE/lib or /opt/splicemachine/lib